src/styles/blocks/product__card.scss

```
@@ -0,0 +1,90 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  background: $light-color;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    display: flex;
+    align-self: center;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin-top: 40px;
+    line-height: 18px;
+    font-weight: 500;
+    font-size: 12px;
+    color: $second-color;
+  }
+
+  &__id {
+    margin: 4px 0 16px;
+    font-weight: 400;
+    font-size: 10px;
+    color: $grey-color;
+    line-height: 14px;
+  }
+
+  &__reviews {
+    display: flex;
+    height: 16px;
+    align-items: center;
+    justify-content: space-between;
+    padding: 0;
+  }
+
+  &__rewiew-word {
+    font-weight: 400;
+    font-size: 10px;
+    color: $second-color;
+  }
+
+  &__prace {
+    margin-top: 24px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    &-word {
+      font-size: 12px;
+      font-weight: 400;
+      color: $grey-color;
+
+    }
+
+    &-how-much {
+      line-height: 18px;
+      color: $second-color;
+      font-weight: 700;
+    }
+  }
+
+  &__bottom-link {
+    margin-top: 16px;
+    display: block;
+    text-decoration: none;
+    color: $light-color;
+    padding: 11px 56px;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    border: 1px solid transparent;
+    transition: color 0.3s, background-color 0.3s, border-color 0.3s;
+
+    text-transform: uppercase;
+    background: $base-color;
+    border-radius: 5px;
+
+    &:hover {
+      border-color: $base-color;
+      color: $base-color;
+      border-radius: 5px;
+      background-color: $light-color;
+      transition: 0.4s;
```

You already transition for element. Don't need to add it on hover pseudo-class 
```suggestion
    &:hover {
      border-color: $base-color;
      color: $base-color;
      border-radius: 5px;
      background-color: $light-color;
```